tests/template/srw_resize_3d_test.py

```diff
@@ -0,0 +1,36 @@
+# -*- coding: utf-8 -*-
+"""PyTest for :mod:`sirepo.template.srw`
+
+:copyright: Copyright (c) 2023 RadiaSoft LLC.  All Rights Reserved.
+:license: http://www.apache.org/licenses/LICENSE-2.0.html
+"""
+from pykern.pkcollections import PKDict
+from pykern.pkunit import pkeq, pkok
+
+_N_BINS = 100
+
+
+def setup_module(module):
+    import os
+
+    # Set max to something reasonable for testing
+    os.environ.update(
+        SIREPO_JOB_MAX_MESSAGE_BYTES=str(_N_BINS * _N_BINS),
+    )
+
+
+def test_srw_resize_3d(fc):
+    from sirepo.template import srw
+    import numpy
+
+    a, xr, yr = srw._reshape_3d(
+        numpy.random.rand(srw._MAX_MESSAGE_BYTES),
+        [0, 0, 0, 0.0, 1.0, _N_BINS, 0.0, 1.0, _N_BINS],
+        PKDict(),
+    )
+    pkok(
```

Comment:
For tests, it's better to keep things separate so that you can know which parameter failed.

```py
pkok(xr[2] < _N_BINS, "xr[2]={}", xr[2])
pkok(yr[2] < _N_BINS, "yr[2]={}", yr[2])
```

In general, giving the information this way is more useful, because it is shorter. You have to look at the test to see what the error is, but the text is easy to find.

Reply:
[05493a0](https://github.com/radiasoft/sirepo/pull/6186/commits/05493a02bfb2ad3e47d3ac48ad9082ad242f5665)